src/themes/teams/components/Tree/treeListItemStyles.ts

```diff
@@ -0,0 +1,10 @@
+import { ICSSInJSStyle } from '../../../types'
+
+const treeListItemStyles = {
+  root: (): ICSSInJSStyle => ({
+    listStyleType: 'none',
+    padding: '0 0 0 .5rem',
```

Comment:
please, consider to use `pxToRem` here, as this value, apparently, should scale with page's font size being changed